### DISCUSSION TITLE
Issue with GitHub Copilot Chat Access After Account Change

### DISCUSSION THREAD

**Discussion by ghost** [community/community#109606] [GitHub Education]
### Select Topic Area

Question

### Body

Hello everyone,

I hope you're doing well. I recently encountered an issue with GitHub Copilot Chat after making some changes to my GitHub account, and I was hoping to get some guidance or assistance from the community here.

Previously, my GitHub account was registered under the username "gabrieluwaila" with the email address gabriel.uwaila@student.kdg.be. However, for some reasons, I decided to delete that account and create a new one under the username "gabrielkdg," still using the same email address.

After creating the new account, I noticed that my access to GitHub Copilot Chat was no longer available. This is quite problematic for me, as my previous account had a subscription associated with it, specifically a School account.

I've been trying to figure out how to restore access to GitHub Copilot Chat on my new account "gabrielkdg," but so far, I haven't had any success.

Has anyone else encountered a similar issue or knows how to resolve it? Any advice or assistance would be greatly appreciated.

Thank you all in advance for your help.

Best regards,

Gabriel

**Comment by namith1003** [comment]
Since you have deleted your account completely first thing to do is try this, do it as fast as possible and open a ticket up.

https://docs.github.com/en/site-policy/other-site-policies/github-account-recovery-policy

if still not possible then contact

Ivan Perez: ivan.perezdominguez@nasa.gov

He seems to be the maintainer for problems regarding copilot or maybe go to this and contact the support team,

https://github.com/enterprise/contact?scid=&utm_campaign=2023q3-site-ww-PremiumSupport&utm_content=Premium+Support&utm_medium=referral&utm_source=github

cause this doesnt seem like an issue easy to revert and since the account you had the copilot purchased on is now deleted you may need to keep the receipt and as much proof and documents as you can when contacting them, may help you a lot to prove your identity and your purchase.

Hope this helps you out and hope this issue gets sorted soon.

**Comment by kher4139** [comment]
Similar issue as above where I have renamed my GitHub username and I no longer see model picker options + Ask/Edit features for my github copilot licensed account , anyone faced similar issue or any resolution?
